src/imitation/util/logger.py

```diff
@@ -108,10 +127,12 @@ def accumulate_means(self, subdir: types.AnyPath) -> Generator[None, None, None]
         if self.current_logger is not None:
             raise RuntimeError("Nested `accumulate_means` context")
 
+        name = types.path_to_str(name)
+        subdir = os.path.join(*self._prefixes, name)
```

Comment:
Again, pathlib is the reccommended way to handle this IMO (I'm sure there's a ton of other places where os.path is used and if we were to use pathlib more generally it should probably be addressed in a separate PR.)

Reply:
Issue opened regarding path lib.